src/platform-includes/getting-started-verify/javascript.nextjs.mdx

```diff
@@ -38,6 +38,6 @@ export default withSentry(handler);
 
 <Note>
 
-Errors triggered from within Browser DevTools are sandboxed, so will not trigger an error handler. Place the snippet directly in your code instead.
```

Comment:
You can't paste the snippets above in the console --> this sentence doesn't make sense.

Also, why does the user care about an error handler not being triggered? They most likely care about error monitoring not being triggered --> reworded